server/src/main/java/org/candlepin/guice/CandlepinContextListener.java

```diff
@@ -224,6 +228,21 @@ private void initializeSubsystems(Injector injector) throws Exception {
             capabilities.add(CandlepinCapabilities.KEYCLOAK_AUTH_CAPBILITY);
         }
 
+        // Init CRL file
+        this.crlFileUtil = injector.getInstance(CrlFileUtil.class);
+        String filePath = getCrlFilePath();
+        File crlFile = new File(filePath);
+        if (!crlFile.exists() || crlFile.length() == 0) {
+            try {
+                this.crlFileUtil.initializeCRLFile(crlFile, Collections.emptyList());
+            }
+            catch (IOException e) {
+                log.error("Error occurred during initialization of CRL file!", e);
+                throw new ConfigurationException("Error occurred during initialization of CRL file!", e);
```

Comment:
This one should stay as an IOException, just rethrown.

Reply:
Done.